libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfigurationForm.tsx

```diff
@@ -201,15 +205,19 @@ const NetworkConfigurationPage = ({ cluster }: { cluster: Cluster }) => {
       };
 
       if (params.userManagedNetworking) {
+        params.apiVips = [];
```

Comment:
I will look into the effect, but sending empty strings and arrays instead of deleting the entries is correct. (https://issues.redhat.com/browse/MGMT-15117)

Reply:
In that case I don't know why the API is rejecting that request, because as you can see in the screenshot, the api_vip(s) fields are either empty or an empty array when UMN is selected.

In the effect, I would expect to be setting both fields for api_vip and api_vips.

Reply:
I believe that was a BE issue which has since been fixed. I have updated my BE today and it's been working fine since then.